contrib/stack/topsStack/FilterAndCoherence.py

```diff
@@ -68,7 +68,7 @@ def runFilter(infile, outfile, filterStrength):
 def runFilter_gaussian(infile, outfile, filterStrength):
     from isceobj import Filter
     
-    logger.info("Applying power-spectral filter")
+    logger.info("Applying gaussian filter")
```

Comment:
The filter is not Gaussian. It is the power-spectral adoptive filter.

Reply:
Thank you @hfattahi for the clarification, it's now rolled back to the original accurate msg.